Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -388,6 +388,8 @@ def set_train_data(self, data):
             self.Warning.missing_data(self._which_missing_data())
             if data:
                 data = HasClass()(data)
+                if not len(data):
+                    data = None
```

Comment:
Is the warning still OK in this case? I do not know what it is, but we need to show clearly that nothing can get computed.

Reply:
When there is no target variable it shows an error: "Train data input requires a target variable." and now where there is a target variable with only `nan` values it shows a warning: "Instances with unknown target values were removed from train data.".

What would be the best error / warning message?

Reply:
@markotoplak : Now I fixed it in a different way.